entries/capman/game.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Isn't this a duplicate? (I can see you also edited `Capman` with capital C)

Reply:
done!